apps/docs/docs/learn/02-thinking-in-stylex.mdx

```diff
@@ -249,15 +249,15 @@ type Props = {
 ```
 
 Styles being typed enables extremely sophisticated rules about the ways in which
-a component styles can be customized with **zero-runtime cost**.
+a component's styles can be customized with **zero-runtime cost**.
 
 ### Shareable constants
 
-CSS class names, CSS variables and other CSS identifiers are defined in a global
-namespace. The obvious and common API to bring them into JavaScript is as
+CSS class names, CSS variables, and other CSS identifiers are defined in a global
+namespace. The obvious and common API to bring them into JavaScript are as
 strings. However, this means losing type-safety and composability.
```

Comment:
I think it would be better to reword this sentence, for example, "Bringing CSS strings into JavaScript can be losing type-safety and composability"

Reply:
agreed, makes sense to me